synapse/cortex.py

```diff
@@ -6282,13 +6284,14 @@ def _loadCoreModule(self, ctor, conf=None):
             logger.exception('mod load fail: %s' % (ctor,))
             return None
 
-    async def getPropNorm(self, prop, valu):
+    async def getPropNorm(self, prop, valu, typeopts=None):
```

Comment:
This one was just to keep it in line with getTypeNorm. It's not strictly necessary but felt odd to exclude it since these two feel like a matched pair.